src/store/plugins/names.js

```diff
@@ -125,7 +125,7 @@ export default (store) => {
       async setDefaults(
         { rootGetters: { activeNetwork, accounts }, commit },
       ) {
-        await Promise.all(accounts.map(async ({ address }) => {
+        await Promise.allSettled(accounts.map(async ({ address }) => {
```

Comment:
is there any difference between `Promise.all` and `Promise.allSettled` except for the return value? I would use `Promise.all` by default